charts/rekor/templates/server/ingress.yaml

```diff
@@ -10,18 +10,24 @@ metadata:
 {{ toYaml .Values.server.ingress.annotations | indent 4 }}
 spec:
   rules:
-    - host: {{ required "An Ingress hostname is required" .Values.server.ingress.hostname }}
+    {{- range .Values.server.ingress.hosts }}
```

Comment:
it is missing the ingress.hosts in the values.yaml and also will need to add it in `ci/ci-values.yaml` (maybe need to remove the exiting hostname)

Reply:
added ingress.hosts, thanks for pointing that out.
updated ci/ci-values.yaml

ptal